vod/media_format.h

```diff
@@ -14,7 +14,14 @@
 // constants
 #define MAX_CODEC_NAME_SIZE (64)
 #define MAX_FRAME_SIZE (10 * 1024 * 1024)
-#define MAX_TRACK_COUNT (1024)
+#ifdef NGX_VOD_MAX_TRACK_COUNT
+#define MAX_TRACK_COUNT (NGX_VOD_MAX_TRACK_COUNT)
+#else
+#define MAX_TRACK_COUNT (64)
+#endif
+#if MAX_TRACK_COUNT % 64 != 0
```

Comment:
would prefer to put it inside the #ifdef (even though it makes no difference...)